discovery-provider/plugins/notifications/src/email/notifications/components/Body.tsx

```diff
@@ -75,7 +75,7 @@ const snippetMap = {
     const suffix = getNumberSuffix(rank)
     return `Your Track ${notification.entity.title} is ${notification.rank}${suffix} on Trending Right Now!`
   },
-  ['createTrack'](notification) {
+  ['create'](notification) {
     const [user] = notification.users
     if (notification.entity.type === 'track' && !isNaN(notification.entity.count) && notification.entity.count > 1) {
       return `${user.name} released ${notification.entity.count} new ${notification.entity.type}`
```

Comment:
super nit - should we add an `s` to `${notification.entity.type}` to make it plural? since its multiple items